src/Word2vec/index.js

```diff
@@ -19,7 +17,7 @@ class Word2Vec {
       .then(response => response.json())
       .then((json) => {
         Object.keys(json.vectors).forEach((word) => {
-          this.model[word] = Array1D.new(json.vectors[word]);
+          this.model[word] = tensor(json.vectors[word]);
```

Comment:
The docs specify `tensor1D()`, `tensor2d()`, etc. are preferred.

> In general, we recommend using this method as it makes code more readable

This probably comes from the codebase being in typsecript but maybe makes sense to carry over for readability?

Reply:
Sounds good, I'll update all reference to scalars.

Reply:
updated!